app/models/service_ansible_playbook.rb

```diff
@@ -1,17 +1,74 @@
 class ServiceAnsiblePlaybook < ServiceGeneric
+  # A chance for taking options from automate script to override options from a service dialog
+  def preprocess(action, add_options = {})
+    save_job_options(action, add_options)
+  end
+
   def execute(action)
-    _log.info("Execute for Service context: #{action}")
+    jt = job_template(action)
+    opts = get_job_options(action)
+
+    _log.info("Launching Ansible Tower job with options: #{opts}")
```

Comment:
There is a plan to enhance this part to address encryption of password. Another PR will follow.